### PR TITLE
Temp solution for pulling recipes from nei to AE patterns

### PR DESCRIPTION
**Pull request by RhyVis**
Move one recipe to `postInit` to init recipemap earlier, sorry about my poor knowledge on java if there is a more convenient solution. At least it works for me with all recipes under GTCMRecipeMap.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -28,27 +28,12 @@ import java.util.concurrent.TimeUnit
 
 buildscript {
     repositories {
-        mavenCentral()
```
Comment (Nxer) [inline build.gradle:31]: ？

File: build.gradle [hunk 2]
```diff
@@ -28,27 +28,12 @@ import java.util.concurrent.TimeUnit
...
-            name 'forge'
-            url 'https://maven.minecraftforge.net'
-        }
         maven {
             // GTNH RetroFuturaGradle and ASM Fork
             name "GTNH Maven"
             url "http://jenkins.usrv.eu:8081/nexus/content/groups/public/"
             allowInsecureProtocol = true
         }
-        maven {
```
Comment (Nxer) [inline build.gradle:43]: do not change this file

File: src/main/java/com/Nxer/TwistSpaceTechnology/recipe/machineRecipe/MiracleTopRecipePool.java [hunk 3]
```diff
@@ -1251,10 +1254,7 @@ public void loadRecipes() {
             .eut(RECIPE_MAX)
             .duration(20 * 1919810)
             .addTo(MT);
-
-
         // endregion
-        loadMaxRecipe();
```
Comment (Nxer) [inline src/main/java/com/Nxer/TwistSpaceTechnology/recipe/machineRecipe/MiracleTopRecipePool.java:1257]: process a mistake when resolving conflicts

**Review by Nxer — CHANGES_REQUESTED** [review]
Loading recipes in completingInit is because core mod init items in postInit and cause some error when all items.
If these recipes are using core mod's item it will cause error.
If no error it is ok.

File: build.gradle (hunk 2 above)
Comment (RhyVis) [reply build.gradle:43]: bs got updated somehow during tests, I'll revert it

**Review by Nxer — APPROVED** [review]

**Pull request merged**